website/content/api-docs/secret/transform.mdx

```diff
@@ -1071,16 +1093,16 @@ This endpoint decodes the provided value using a named role.
   applicable for FPE transformations with `supplied` or `generated` as the tweak
   source. The tweak must be a 7-byte value that is then base64 encoded.
 
-- `reference` `(string: "")` - 
+- `reference` `(string: "")` -
   Only valid on batch requests using 'batch_input' below.  A user supplied
-  string that will be present on the corresponding item in the batch 
+  string that will be present on the corresponding item in the batch
   response, to assist in correlating inputs with outputs.
 
 - `batch_input` `(array<object>: nil)` -
   Specifies a list of items to be encoded in a single batch. When this
```

Comment:
Shouldn't this be

Specifies a list of items to be **decoded** in a single batch

Reply:
Good catch, I'll push an update with a correction.